aerogear/mobile-core
# Add a new role to provision mobile core on rhpds II

Based on https://github.com/aerogear/mobile-core/pull/125
and following these guidelines https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/setup-mobile-on-existing-cluster.adoc

To verify:

Provision a openshift instance from RHPDS (can use this one: ssh -i ocp-workshop.pem ec2-user@bastion.mogrodni.openshiftworkshop.com, I will email you the private key)
SSH into the workstation, and run the following command:
```
sudo bash
git clone https://github.com/witmicko/mobile-core.git
git checkout aerogear-mobile-role
ansible-playbook installer/rhpds-playbook.yml
```

Web Console - https://master.mogrodni.openshiftworkshop.com/console/catalog?category=all
ping me for username/password
When it is finished, the mobile tab should be available in the catalog
